packages/api/src/validation/apiValidator.ts

```diff
@@ -1,8 +1,10 @@
 export type APIValidatorRequest = {
   method: string;
   path: string;
-  query?: unknown;
-  body?: unknown;
+  contentType?: string;
+  params?: { [key: string]: unknown };
+  query?: { [key: string]: unknown };
+  body?: any;
```

Comment:
Just curious: can't keep `unknown` here?

Reply:
Oops, yes we can. I'll revert this change.